largest_palindrome_product.js

```diff
@@ -5,18 +5,69 @@
  *  @return {Object} an object containing the two factors used to produce
  *                   the palindromeNumber and the palindromeNumber itself.
  */
-module.exports = function(digits){
+module.exports = function (digits) {
   var factor_0 = 0;
   var factor_1 = 0;
+  var dummyfactor0 = 0;
+  var dummyfactor1 = 0
   var palindromeNumber = 0;
+  let product = 0;
+  let max = 0;
 
   // do your work here
 
+  function palindromeCheck(input) {
+    var palindrome = parseInt(
+      input
```

Comment:
Did your formatter do this? It's usually best to make it into just one line.

Reply:
Ok, it did. I will turn it off so it doesn't do that.